tests/ProfileBrowserTest.php

```diff
@@ -73,27 +86,39 @@ public function testPageNumbers()
 
     /**
      * Checks left pagination arrow appears when it should
-     * 
+     *
      * (It should be hidden for page 1 and not for other pages)
      */
     public function testLeftArrow()
     {
-        // @todo Finish this
-        $this->markTestIncomplete(
-            'This test has not been implemented yet.'
-        );
+        factory(Profile::class, 'withAUser', 36)->create();
+        $this->visit(route('profile.browse'))
+            ->dontSeeInElement('pagination', tran('pagination.prev'));
+        $this->visit(route('profile.browse', ['page'=>3]))
+            ->seeInElement('pagination', tran('pagination.prev'));
     }
 
     /**
      * Checks right pagination arrow appears when it should
-     * 
+     *
      * (It should be hidden for the last page and not for other pages)
      */
     public function testRightArrow()
     {
-        // @todo Finish this
-        $this->markTestIncomplete(
-            'This test has not been implemented yet.'
-        );
+        factory(Profile::class, 'withAUser', 36)->create();
+        $this->visit(route('profile.browse'))
+            ->seeInElement('pagination', tran('pagination.next'));
+        $this->visit(route('profile.browse', ['page'=>3]))
+            ->dontSeeInElement('pagination', tran('pagination.next'));
+    }
+
+    /**
+     * Migrates the database and set the mailer to 'pretend'.
+     * This will cause the tests to run quickly.
+     */
+    private function prepareForTests()
```

Comment:
If this function is gonna be used in almost all the test, as I think that probably will, probably it should be in the parent class (TestCase) so we don't have to call it every time.